platform/osx/os_osx.mm

```diff
@@ -390,8 +401,13 @@ - (NSRect)firstRectForCharacterRange:(NSRange)aRange actualRange:(NSRangePointer
 
 - (void)cancelComposition {
 	[self unmarkText];
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 1070
```

Comment:
Our minimum support version is 10.9 according to `detect.py`, so I'm not sure this is necessary.